scripts/windows/sudo-install-wsl.ps1

```diff
@@ -0,0 +1,18 @@
+# //////////////////////////////////////////////////////////////////////
+# sudo-install-ws1.ps1
+
+Param(
+    [Parameter(Mandatory)]
+    [ValidateSet("EnableVMPlatform-02", "InstallLinuxUpdatePackage-03")]
```

Comment:
This is missing `EnableWSL-01`, which was set in: https://github.com/rancher-sandbox/rancher-desktop/pull/197/files#diff-4b32ed147499b162d65f9113cd2f0f9ab2d9c3d6b9615e3aada796b95d48d3a0R86

Reply:
That's because it isn't needed -- the user would normally kick this off running as administrator with the command-line `scripts\windows\install-wsl.ps1` .  `sudo-install-ws1.ps1` is a helper file used internally (maybe it should start with an `_` then). You think I should add the first step to sudo just for completeness?

Reply:
Should be fixed now ... the fix supports running `sudo-install-wsl.ps1 -Step "EnableWSL-01"`